### PR TITLE
haz-db-origin-consumer

### LINKED ISSUE #93: add a db consumer for origin
Add a db consumer app for the origin web site.  This should replicate the db functionality of mule-quake-client.

It will need to use the qrt schema etc.

### PR DESCRIPTION
**Pull request by gclitheroe**
Adds a db quake consumer for use with the origin (geonet) web servers.

resolves #93.

I've tested this code (especially db.SaveQuakeQRT) using the database schema from the geonet project.  I loaded 11000+ SC3ML files from 2015 using haz-db-loader-origin.  Worked ok.

This will let me move the origin servers to AWS (and get rid of mule-quake-client) in the process.

### CODE REVIEW

File: database/save_quake.go [hunk 1]
```diff
@@ -216,3 +216,48 @@ func (db *DB) SaveHeartBeat(h msg.HeartBeat) error {
...
+		txn.Rollback()
+		return err
+	}
+
+	return txn.Commit()
+}
+
+// SaveHeartBeatQRT saves heartbeat messages into the QRT schema.  This is used by
+// the origin web servers.  This func can be removed once the web site is
+// using the API.
```
Comment (junghao) [inline database/save_quake.go]: The comment here needs to update.

File: README.md [hunk 2]
```diff
@@ -57,6 +57,18 @@ cd haz-db-loader
 godep go run haz-db-loader.go
```
Comment (bpeng) [inline README.md]: godep is no longer used

File: haz-db-origin-loader/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+Download some SC3ML using the aws cli (the bucket should be publicly accessible for read) e.g.,
+
+```
+aws s3 sync s3://seiscompml07 /work/seismcompml07 --exclude "*"  --include "2015p*"
+```
+
+Build and run the db loader.
+
+```
+godep go build && ./haz-db-origin-loader
```
Comment (bpeng) [inline haz-db-origin-loader/README.md]: godep is no longer used

**Comment by junghao** [comment]
Everything is ok. Just need to update comments and readme.

**Comment by bpeng** [comment]
code looks clear, tested ```haz-db-origin-loader``` OK

**Pull request merged**